client/client.go

```diff
@@ -275,7 +275,7 @@ func NewClient(svrAddrs []string, security SecurityOption, opts ...ClientOption)
 // NewClientWithContext creates a PD client with context.
 func NewClientWithContext(ctx context.Context, svrAddrs []string, security SecurityOption, opts ...ClientOption) (Client, error) {
 	log.Info("[pd] create pd client with endpoints", zap.Strings("pd-address", svrAddrs))
-	c, clientCtx, clientCancel, tlsCfg := createClient(ctx, &security)
+	c, clientCtx, clientCancel, tlsCfg := createClient(ctx, 0, &security)
```

Comment:
Please add a comment to explain that `0` represents the default group.

Reply:
added